src/modules/navigator/rtl_params.c

```diff
@@ -113,6 +113,8 @@ PARAM_DEFINE_FLOAT(RTL_MIN_DIST, 5.0f);
  * @value 0 Return home via direct path
  * @value 1 Return to a planned mission landing, if available, via direct path, else return to home via direct path
  * @value 2 Return to a planned mission landing, if available, using the mission path, else return to home via the reverse mission path
+ * @value 3 Return via direct path to closest safe point (rally point), if available, else return to home via direct path
```

Comment:
I would merge this into option 0, because:
- A user who wants safe points would need to change the parameter, adding one more configuration hurdle.
- For a user who does not want safe points it still works, since they would simply not configure any safe points.

Reply:
@dagar you mentioned last week during the devcall that you'd prefer the Rally Points not enabled with the RTL_TYPE set to 0 (default). I kind of prefer what beat proposes, to enable landing at Rally Points as soon as there are some planned. 

Actually the consequent way would then be to also remove the RTL_TYPE 4 I added, as this then would be the same as option 1 (It would land at home via mission landing, but at the rally points without).

Of course we could also bring it in with all 5 RTL_TYPEs to have the whole Rally Point feature a bit more hidden, and then change to the original 3 RTL_TYPES when we have different RTL actions depending on what triggered it (eg not go to safe landing for RC loss).